code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -955,14 +955,19 @@
 	buckle_mob_flags= RIDER_NEEDS_ARM // just in case
 	return ..()
 
+/mob/living/silicon/robot/can_resist()
+	if(lockcharge)
+		balloon_alert(src, "locked down!")
+		return FALSE
+	. = ..()
```

Comment:
```suggestion
	return ..()
```

basically this just means the same exact thing compiler-wise but it's nice to have it like this because it's more explicit that the proc terminates after this point

Reply:
Got it, thanks!